style.css

```diff
@@ -4,6 +4,297 @@
+    color: #777;
+    padding-right: 15px;
+    padding-left: 15px;
+}
+.video-icon{
+    margin-left: auto;
+}
+#extension{
+    justify-self: center;
+    width: 85px;
+    height: 30px;
+    margin-left: 20px;
+    /* padding-bottom: 10px; */
+}
+#channel{
+    margin-top: 15px;
+    display: flex;
+    justify-items: center;
+}
+#channel-logo{
+    width: 55px;
+    height: 55px;
+    border-radius: 50%;
+}
+.channel-name{
+    display: flex;
+    padding: 10px;
+}
+.channel-name{
+    flex-direction: column;
+}
+#channel .sub-number{
+    font-size: 15px;
+    padding-top: 3px;
+}
+.btn-sub{
+    height: 40%;
+    margin-left: auto;
+    align-items: center;
+    margin-top: 10px;
+}
+.btn{
+    font-size: 15px;
+    color: #fff;
+    background:rgba(248, 21, 4, 0.959);
+    padding: 10px 20px;
+    border: none;
+    font-weight: bold;
+    border-radius: 3px;
+    cursor: pointer;
+}
+.comment{
+    margin: auto;
+    margin-top: 20px;
+    width: 85%;
+    line-height: 1.5rem;
+    margin-bottom: 20px;
+}
+.btn-showmore{
+    font-size: 15px;
+    color: grey;
+    background: transparent;
+    border: none;
+    cursor: pointer;
+    margin-left: auto;
+    padding-top: 15px;
+    font-weight: bold;
+}
+.comment-extra{
+    margin-top: 20px;
+    margin-bottom: 20px;
+    display: flex;
+    font-size: 14px;
+    font-weight: 450;
+}
+.comment-extra i{
+    margin-left: 5%;
+    margin-right: 1%;
+    font-size: 20px;
+    color: #777;
+}
+.comment-section{
+    display: flex;
+}
+#comment-search{
+    width: 100%;
+    border: none;
+    margin-left: 20px;
+    border-bottom: solid 1px gray;
+    padding-top: 10px;
+    margin-bottom: 20px;
+    outline: none;
+    background-color: transparent;
+}
+.comments{
+    margin-top: 25px;
+    display: flex;
+}
+.comments .comment-writup{
+    padding: 7px;
+}
+.bold{
+    font-weight: bold;
+    padding-bottom: 7px;
+}
+.comment-des{
+    padding-top:5px;
+    padding-bottom: 7px;
+}
+.comment-icon{
+    display: flex;
+    font-size: 20px;
+    color: #777;
+    margin-top: 10px;
+}
+.comment-icon p{
+    padding-right: 30px;
+    margin-right: 10px;
+}
+
+/* Aside Bar */
+#sidebar{
+    width: 30%;
+}
+.next{
+    display: flex;
+}
+.next i{
+    font-size: 20px;
+    color: rgb(3, 24, 117);
+    margin-left: 20px;
+}
+.auto123{
+    margin-left: auto;
+    font-size: 16px;
+}
+.up{
+    font-size: 16px;
+}
+#imagethumbnail{
+    display: flex;
+    margin-top: 15px;
+    margin-bottom: 15px;
+}
+.thumbnail-style{
+    width: 190px;
+    height: 105px;
+}
+.thumbnail-title, p{
+    font-size: 14px;
+    padding-left: 5px;
+}
+.thumb-bold{
+    font-weight: 550;
+    padding-bottom: 10px;
+    width: 220px;
+}
+.thum-name{
+    padding-bottom: 5px;
+    color: gray;
+}
+.thum-view{
+    color: gray;
+}
+.left{
+    margin-left: auto;
+    padding-top: 15px;
+    font-size: 20px;
+    color: #777;
+}
+.left2{
+    margin-left: auto;
+    font-size: 20px;
+    color: #777;
+}
```

Comment:
You have a lot of repeated blocks of CSS declarations. [check this article](https://speckyboy.com/good-bad-css-practices/) for more on good and bad practices of CSS